share/pot/destroy.sh

```diff
@@ -61,6 +61,10 @@ _pot_zfs_destroy()
 		_error "zfs failed to destroy the dataset $_jdset"
 		return 1 # false
 	fi
+	if [ -d "${POT_FS_ROOT}/jails/$_pname" ]; then
```

Comment:
While probably it's a good idea to have those kind of robust clean-up features, we should aim to not depend on them for "normal" use cases.
I started a patch to fix the cases we introduced that broke `destroy`, I'll submit it later.

Reply:
Yeah, basically rolling back the commit that introduced the mkdir wrapper, that was a good example of failing in a team 😁

I'll start working on the real solution to the permission issue like discussed soonish (essentially `chmod 750 /opt/pot; chown root:pot /opt/pot`)